### PR TITLE
allow a user to change default keybindings

### LINKED ISSUE #6878: not possible to change Monaco keybindings
Monaco keybinding service reacts on default keybindings, even if we change in Theia. So a user don't have a way to change such keybindings. We should replace this service with no-op implementation.

Originally reported in Gitpod: https://github.com/gitpod-io/gitpod/issues/1060

### PR DESCRIPTION
**Pull request by akosyakov**
#### What it does

- fix #6878: allow a user to change default keybindings

#### How to test

- change some editor default keybinding, like rename symbol
- check that default keybinding does not work anymore

#### Review checklist

- [x] as an author, I have thoroughly tested my changes and carefully followed [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#requesting-a-review)

#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

### CODE REVIEW

**Comment by akosyakov** [comment]
@spoenemann could you try again please? I've addressed your issues.

File: packages/monaco/src/browser/monaco-editor-provider.ts [hunk 1]
```diff
@@ -144,6 +152,39 @@ export class MonacoEditorProvider {
         return editor;
     }
 
+    /**
+     * Suppresses Monaco keydown listener to avoid triggering default Monaco keybindings
+     * if they are overriden by a user. Monaco keybindings should be registered as Theia keybindings
+     * to allow a user to customize them.
+     */
+    protected suppressMonaconKeybindingListener(editor: MonacoEditor): void {
```
Comment (spoenemann) [inline packages/monaco/src/browser/monaco-editor-provider.ts]: `Monacon` => `Monaco`

**Comment by akosyakov** [comment]
@vince-fugnitto It should be fixed.

**Comment by akosyakov** [comment]
@vince-fugnitto Did it work for you now?

**Pull request merged**